mappings/net/minecraft/enchant/RiptideEnchant.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_3590 net/minecraft/enchant/RiptideEnchant
```

Comment:
Should be moved to `net/minecraft/enchantment` where the rest of the enchantments are and should be renamed to `RiptideEnchantment` for consistency with the other enchantments

Reply:
fixed in 0bb6d26